### ISSUE TITLE
Pending channel stuck on waiting_close

### ISSUE THREAD

**Issue by JPyke3** [lightningnetwork/lnd#4890]
### Background

I have a channel that seems to have hung on `waiting_close`. Here is my output of `lncli pendingchannels:

```
    "waiting_close_channels": [
        {
            "channel": {
                "remote_node_pub": "03634bda49c9c42afd876d8288802942c49e58fbec3844ff54b46143bfcb6cdfaf",
                "channel_point": "ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1",
                "capacity": "442226",
                "local_balance": "433176",
                "remote_balance": "0",
                "local_chan_reserve_sat": "4422",
                "remote_chan_reserve_sat": "4422",
                "initiator": "INITIATOR_LOCAL",
                "commitment_type": "STATIC_REMOTE_KEY"
            },
            "limbo_balance": "433176",
            "commitments": {
                "local_txid": "00d7e9d018c6335bb51c246bd1c19640e72bc371c49d81352f2a11fba4eff2aa",
                "remote_txid": "48f3c1a0f8c91f35e1f55ac205b8cdf11fb66d5754872f86d5a9120f4a8c277d",
                "remote_pending_txid": "",
                "local_commit_fee_sat": "9050",
                "remote_commit_fee_sat": "9050",
                "remote_pending_commit_fee_sat": "0"
            }
        }
    ]
```

This issue looks eerily similar to this: https://github.com/lightningnetwork/lnd/issues/4284

Here is the output of `lnd` grepping for the `channel_point` and this is all I got:

```
Dec 31 14:38:03 raspberrypi lnd[19127]: 2020-12-31 14:38:03.809 [DBG] CNCT: ChannelArbitrator(ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1): new block (height=663845) examining active HTLC's
Dec 31 14:38:03 raspberrypi lnd[19127]: 2020-12-31 14:38:03.809 [DBG] CNCT: ChannelArbitrator(ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1): checking commit chain actions at height=663845, in_htlc_count=0, out_htlc_count=0
Dec 31 14:44:28 raspberrypi lnd[19127]: 2020-12-31 14:44:28.224 [DBG] CNCT: ChannelArbitrator(ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1): new block (height=663846) examining active HTLC's
Dec 31 14:44:28 raspberrypi lnd[19127]: 2020-12-31 14:44:28.224 [DBG] CNCT: ChannelArbitrator(ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1): checking commit chain actions at height=663846, in_htlc_count=0, out_htlc_count=0
Dec 31 14:55:37 raspberrypi lnd[19127]: 2020-12-31 14:55:37.206 [DBG] CNCT: ChannelArbitrator(ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1): new block (height=663847) examining active HTLC's
Dec 31 14:55:37 raspberrypi lnd[19127]: 2020-12-31 14:55:37.206 [DBG] CNCT: ChannelArbitrator(ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1): checking commit chain actions at height=663847, in_htlc_count=0, out_htlc_count=0
Dec 31 14:57:03 raspberrypi lnd[19127]: 2020-12-31 14:57:03.315 [DBG] CNCT: ChannelArbitrator(ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1): new block (height=663848) examining active HTLC's
Dec 31 14:57:03 raspberrypi lnd[19127]: 2020-12-31 14:57:03.315 [DBG] CNCT: ChannelArbitrator(ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1): checking commit chain actions at height=663848, in_htlc_count=0, out_htlc_count=0
```

### Your environment

* version of `lnd`: lnd version `0.11.1-beta commit=v0.11.1-beta`
* which operating system (`uname -a` on *Nix): `Linux raspberrypi 5.4.79-v7l+ #1373 SMP Mon Nov 23 13:27:40 GMT 2020 armv7l GNU/Linux`
* version of `btcd`, `bitcoind`, or other backend: Neutrino
* any other relevant environment details: RasPI 4 4GB

### Steps to reproduce

Unsure how I could reproduce this, I just attempted to close the channel with `lncli closechannel --sat_per_byte 2 ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f 1`

### Expected behaviour

The channel should have gracefully closed

### Actual behaviour

Ongoing

**Comment by JPyke3** [comment]
I see now my system time is out, as well as the other issue I referenced also had his time out (Unless he waited like half a year to post the issue). Going to fix that up and restart `lnd` as well as the pi to see if that sorts my issue out.

**Comment by JPyke3** [comment]
Okay after giving `lnd` the good ol' reboot. Here is my journal logs related the `channel_point`:
```
Dec 31 15:20:41 raspberrypi lnd[19127]: 2020-12-31 15:20:41.553 [DBG] CNCT: ChannelArbitrator(ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1): new block (height=663849) examining active HTLC's
Dec 31 15:20:41 raspberrypi lnd[19127]: 2020-12-31 15:20:41.553 [DBG] CNCT: ChannelArbitrator(ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1): checking commit chain actions at height=663849, in_htlc_count=0, out_htlc_count=0
Dec 31 15:27:53 raspberrypi lnd[19127]: 2020-12-31 15:27:53.160 [DBG] CNCT: Stopping ChannelArbitrator(ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1)
Dec 31 15:29:10 raspberrypi lnd[20911]: 2020-12-31 15:29:10.952 [DBG] CHBU: Crafting backup for ChannelPoint(ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1)
Dec 31 15:29:11 raspberrypi lnd[20911]: 2020-12-31 15:29:11.993 [DBG] CNCT: New ChainEventSubscription(id=0) for ChannelPoint(ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1)
Dec 31 15:29:11 raspberrypi lnd[20911]: 2020-12-31 15:29:11.994 [WRN] CNCT: Channel ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1 is in state ChanStatusCoopBroadcasted, but no coop closing tx to re-publish...
Dec 31 15:29:11 raspberrypi lnd[20911]: 2020-12-31 15:29:11.994 [DBG] CNCT: Starting chain watcher for ChannelPoint(ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1)
Dec 31 15:29:11 raspberrypi lnd[20911]: 2020-12-31 15:29:11.994 [DBG] NTFN: Using height hint 663849 retrieved from cache for outpoint=ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1, script=0 40d898e5b4182b585eb04f6e433d93399395ddd6008fdc4602bd094307296b93 instead of 663543
Dec 31 15:29:11 raspberrypi lnd[20911]: 2020-12-31 15:29:11.995 [INF] NTFN: New spend subscription: spend_id=1, outpoint=ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1, script=0 40d898e5b4182b585eb04f6e433d93399395ddd6008fdc4602bd094307296b93, height_hint=663849
Dec 31 15:29:11 raspberrypi lnd[20911]: 2020-12-31 15:29:11.995 [INF] NTFN: Dispatching historical spend rescan for outpoint=ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1, script=0 40d898e5b4182b585eb04f6e433d93399395ddd6008fdc4602bd094307296b93, start=663849, end=663849
Dec 31 15:29:11 raspberrypi lnd[20911]: 2020-12-31 15:29:11.995 [DBG] BTCN: Enqueuing request for ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1 with birth height 663849
Dec 31 15:29:11 raspberrypi lnd[20911]: 2020-12-31 15:29:11.996 [INF] CNCT: Close observer for ChannelPoint(ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1) active
Dec 31 15:29:11 raspberrypi lnd[20911]: 2020-12-31 15:29:11.998 [DBG] CNCT: Starting ChannelArbitrator(ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1), htlc_set=(map[contractcourt.HtlcSetKey]contractcourt.htlcSet) (len=2) {
Dec 31 15:29:12 raspberrypi lnd[20911]: 2020-12-31 15:29:11.998 [INF] CNCT: ChannelArbitrator(ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1): starting state=StateDefault, trigger=chainTrigger, triggerHeight=663849
Dec 31 15:29:12 raspberrypi lnd[20911]: 2020-12-31 15:29:11.998 [DBG] CNCT: ChannelArbitrator(ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1): new block (height=663849) examining active HTLC's
Dec 31 15:29:12 raspberrypi lnd[20911]: 2020-12-31 15:29:11.998 [DBG] CNCT: ChannelArbitrator(ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1): checking commit chain actions at height=663849, in_htlc_count=0, out_htlc_count=0
Dec 31 15:29:12 raspberrypi lnd[20911]: 2020-12-31 15:29:11.999 [DBG] CNCT: ChannelArbitrator(ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1): new block (height=663849) examining active HTLC's
Dec 31 15:29:12 raspberrypi lnd[20911]: 2020-12-31 15:29:12.007 [DBG] CNCT: ChannelArbitrator(ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1): checking commit chain actions at height=663849, in_htlc_count=0, out_htlc_count=0
Dec 31 15:29:12 raspberrypi lnd[20911]: 2020-12-31 15:29:12.057 [DBG] LNWL: ChannelPoint(ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1): starting local commitment: (*lnwallet.commitment)(0x2432000)({
Dec 31 15:29:12 raspberrypi lnd[20911]:     PreviousOutPoint: (wire.OutPoint) ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1,
Dec 31 15:29:12 raspberrypi lnd[20911]: 2020-12-31 15:29:12.068 [DBG] LNWL: ChannelPoint(ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1): starting remote commitment: (*lnwallet.commitment)(0x25045a0)({
Dec 31 15:29:12 raspberrypi lnd[20911]:     PreviousOutPoint: (wire.OutPoint) ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1,
Dec 31 15:29:12 raspberrypi lnd[20911]: 2020-12-31 15:29:12.069 [DBG] LNWL: ChannelPoint(ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1): Restoring 0 dangling remote updates
Dec 31 15:29:12 raspberrypi lnd[20911]: 2020-12-31 15:29:12.070 [DBG] LNWL: ChannelPoint(ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1): Restoring 0 local updates that the peer should sign
Dec 31 15:29:12 raspberrypi lnd[20911]: 2020-12-31 15:29:12.070 [DBG] FNDG: ChannelPoint(ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1) with chan_id=2fb8deb06c0a449c27bbb4c023e9065cf272296a85562db7b1590ca8170c70cb not found in opening database, assuming already announced to the network
Dec 31 15:29:12 raspberrypi lnd[20911]: 2020-12-31 15:29:12.085 [WRN] CHBU: Replacing disk backup for ChannelPoint(ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1) w/ newer version
Dec 31 15:29:12 raspberrypi lnd[20911]: 2020-12-31 15:29:12.662 [DBG] LNWL: ChannelPoint(ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1): starting local commitment: (*lnwallet.commitment)(0x2504d80)({
Dec 31 15:29:12 raspberrypi lnd[20911]:     PreviousOutPoint: (wire.OutPoint) ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1,
Dec 31 15:29:12 raspberrypi lnd[20911]: 2020-12-31 15:29:12.672 [DBG] LNWL: ChannelPoint(ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1): starting remote commitment: (*lnwallet.commitment)(0x24322d0)({
Dec 31 15:29:12 raspberrypi lnd[20911]:     PreviousOutPoint: (wire.OutPoint) ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1,
Dec 31 15:29:12 raspberrypi lnd[20911]: 2020-12-31 15:29:12.673 [DBG] LNWL: ChannelPoint(ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1): Restoring 0 dangling remote updates
Dec 31 15:29:12 raspberrypi lnd[20911]: 2020-12-31 15:29:12.674 [DBG] LNWL: ChannelPoint(ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1): Restoring 0 local updates that the peer should sign
Dec 31 15:29:12 raspberrypi lnd[20911]: 2020-12-31 15:29:12.674 [INF] PEER: NodeKey(03634bda49c9c42afd876d8288802942c49e58fbec3844ff54b46143bfcb6cdfaf) loading ChannelPoint(ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1)
Dec 31 15:29:12 raspberrypi lnd[20911]: 2020-12-31 15:29:12.674 [WRN] PEER: ChannelPoint(ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1) has status ChanStatusCoopBroadcasted|ChanStatusLocalCloseInitiator, won't start.
Dec 31 15:29:13 raspberrypi lnd[20911]: 2020-12-31 15:29:13.899 [DBG] BTCN: Adding outpoint=ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1 height=663849 to watchlist
Dec 31 15:29:13 raspberrypi lnd[20911]: 2020-12-31 15:29:13.981 [DBG] BTCN: Outpoint ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1 not found in block 663849
Dec 31 15:29:14 raspberrypi lnd[20911]: 2020-12-31 15:29:14.018 [DBG] NTFN: Updated spend hint to height=663849 for unconfirmed spend request outpoint=ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f:1, script=0 40d898e5b4182b585eb04f6e433d93399395ddd6008fdc4602bd094307296b93
```

Going to wait a few blocks, perhaps it might figure it out. Will post here in either an hour or two if nothing changes, or if it eventually gets through.

*System time matters ladies and gents*

**Comment by JPyke3** [comment]
I saw errors in the journal about co-op negotiating failed. One issue perusal later I came across this: https://github.com/lightningnetwork/lnd/issues/4774#issuecomment-740889794

After giving the transaction a good old `--force` with:
```
lncli closechannel --force --funding_txid ca700c17a80c59b1b72d56856a2972f25c06e923c0b4bb279c440a6cb0deb82f --output_index 1
```

I managed to close my weirdo limbo channel. Issue solved, system time seemed to be the key issue here.

!remindme 20 years when that tx fee is worth more than my house 😂 

Happy new years everyone!
